docs/library/frameworks/template_framework/index.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
```

Comment:
Later, these pages will have to be mentioned somewhere in the guidelines page, where we say "we have here templates ..." so maybe it's better to remove the orphan label, not to forget to link it somewhere. This is just as a tip, I'm not saying it's a better way of organizing the workload :D